en/companion_computer/FlytOS.md

```diff
@@ -0,0 +1,145 @@
+# FlytOS 
+
+[FlytOS](https://flytbase.com) is a software framework which provides Drone APIs and SDKs for building high-level 
+drone applications such as aerial delivery, precision agriculture, surveys, photography, industrial inspections and disaster
+management. It is designed to enable drone-developers build advanced drone applications using its open APIs.
+
+FlytOS is based on Linux and ROS (Robot Operating System), making it an ideal platform for building commercial as well as 
+research orientated drone applications. It supports a wide range of hardware options such as *Raspberry Pi 3, Odroid XU4, 
+Nvidia TX1, Intel Edison, Intel Aero and FlytPOD*. It uses MAVLink to communicate with the autopilot, and exposes high level
+FlytAPIs in *ROS, CPP, Python, REST and Websocket*.
+
+![](../../assets/flytosccsupport.jpg)
+
+
+This makes it easy to build high-level applications using computer-vision, machine-learning, cloud-connectivity and enables
+developers to create their custom user-interfaces on web/mobile devices of their choice. FlytOS also has modules to manage 
+payloads, security and updates. The modular design of FlytOS allows for integration with external ROS/Linux libraries and 
+custom data plumbing between onboard and offboard apps. FlytOS aims to provide a standard language for the drone application
+developers to talk to their drones.
+
+{% youtube %}https://www.youtube.com/watch?v=CZFVWDN5Gcc{% endyoutube %}
+
+
+![](../../assets/FlytOSArch.png)
+   
+
+Developer Tools
```

Comment:
Suggest you don't mix underlining and # for titles. Just use ## for level 2 titles, ### for level 3 etc.